LoginDemo/LearningSelenium/PageModel/Login.cs

```diff
@@ -0,0 +1,84 @@
+﻿using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using static System.Net.Mime.MediaTypeNames;
+
+namespace LearningSelenium.PageModel
+{
+    public class Login
+    {
+        WebDriver driver;
```

Comment:
Next step is read about pattern Singleton and WebDriverManager.
And create an extra class to work with browser and driver. Something like BrowserConfig with such methods as Setup(), GetDriver(), GoToAddress(), GetElement(), GetElements(), etc.

Reply:
Sure will update with Singleton pattern